tests/Feature/Shared/OfferPriceChangedNotificationTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Tests\Feature\Shared;
+
+use App\Models\Offer;
+use App\Models\PriceHistory;
+use App\Models\User;
+use App\Notifications\OfferPriceChangedNotification;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Notification;
+use Tests\TestCase;
+
+class OfferPriceChangedNotificationTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /** @test */
+    public function notification_is_send(): void
```

Comment:
Nie bardzo rozumiem co to testuje, jeśli event to to powinno byc w unit testach. Jeśli zmianę to wydaje mi się że job powinien zostać wywołany, albo request chociaż

Reply:
tu próbowałem ograć sprawdzenie czy wyślą się emaile, zrobię z tego test jednostkowy, do joba zrobię osobny